app/main.py

```diff
@@ -1 +1,84 @@
-# write your code here
+class SoftwareEngineer:
+    def __init__(self, name: str) -> None:
+        self.name = name
+        self.skills = []
+
+    def learn_skill(self, skill: str) -> None:
+        skills_list = skill.split(", ")
+
+        self.skills.extend(skills_list)
+
+
+class FrontendDeveloper(SoftwareEngineer):
+    def __init__(
+            self,
+            name: str,
+    ) -> None:
+        super().__init__(name)
+        self.skills = [
```

Comment:
Here u can do like this self.skills += ["JavaScript", "HTML", "CSS"] or use `extend` to skip rewriting init in FullStackDeveloper class